app/src/androidTest/java/ch/epfl/balelecbud/WelcomeFragmentTest.java

```diff
@@ -1,16 +1,24 @@
 package ch.epfl.balelecbud;
 
 import androidx.fragment.app.testing.FragmentScenario;
+import androidx.test.espresso.matcher.RootMatchers;
 import androidx.test.ext.junit.runners.AndroidJUnit4;
 
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
 import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
 import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.RootMatchers.isPlatformPopup;
+import static androidx.test.espresso.matcher.RootMatchers.withDecorView;
+import static androidx.test.espresso.matcher.ViewMatchers.hasDescendant;
 import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
 import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static junit.framework.TestCase.assertNull;
+import static org.hamcrest.CoreMatchers.not;
```

Comment:
Check your imports, it doesn't seem like you use it all

Reply:
Indeed I don't, I modified this in the new commit